projects/demo/src/modules/components/data-list/examples/2/index.html

```diff
@@ -32,6 +32,7 @@
             Drinks
         </button>
         <button tuiOption>Ice Cream</button>
+        <button tuiOption (click)="closeDropdown()">Close me</button>
```

Comment:
A button to close the dropdown makes little sense. Let's add click handler to each button instead and maybe show notification with something like "You selected {{xxx}}"

Reply:
done